### ISSUE TITLE
Rule cross-ref inside of a rspec hyperlink breaks SLVS

### ISSUE THREAD

**Issue by vnaskos-sonar** [SonarSource/sonarlint-visualstudio#5413]
After the [update of the analyzer](https://github.com/SonarSource/sonarlint-visualstudio/issues/5406) on SLVS it was identified that there is an issue with the following rule due to rule cross-reference being placed inside an `<a>` tag which produces nested hyperlinks during the xaml conversion.

S6962

```
<li> <a href="https://rules.sonarsource.com/csharp/RSPEC-6420/">{rule:csharpsquid:S6420} - Client instances should not be recreated on each Azure
  Function invocation</a> </li>
```

**Comment by martin-strecker-sonarsource** [comment]
This is what it looks like on peach:
![image](https://github.com/SonarSource/sonarlint-visualstudio/assets/103252490/9fbf2256-076c-4587-b9e2-c12ae7e352bc)

**Comment by martin-strecker-sonarsource** [comment]
Will be fixed by SonarSource/rspec/pull/3926

**Comment by vnaskos-sonar** [comment]
Fixed on https://github.com/SonarSource/sonarlint-visualstudio/pull/5510
